index.js

```diff
@@ -7,14 +7,180 @@ function ask(questionText) {
   });
 }
 
-start();
 
-async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+
+let inventory = [] // Creates an array named inventory
+
+class Item { //Create class for Item Object
+  constructor(name, description, action, takeable) {
+    this.name = name,
+      this.description = description,
+      this.action = action || "Nothing happens...",
+      this.takeable = takeable || false;
+  }
+  take() { //Take method allows you to pick items up and add them to your inventory
+    if (this.takeable) {
+      inventory.push(this.name);
+      return `You picked up ${this.name} and stored it in your inventory!`
+    } else {
+      return `You can't take that!`
+    }
+  }
+  use() { //Use method allows you to use items that are inside your inventory
+    if (this.name === "note" && inventory.includes(this.name)) {
+      return this.action
+    } else {
+      return false;
+    }
+  }
+  examineI() { //examine method allows you to examine items
```

Comment:
I think you should be able to use the same examine function for items and rooms to save some typing